src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -503,14 +507,13 @@ def run_single_testcase(self,
     self._test_ssh()
 
     # We need to push the testcase to the device and pass in the name.
-    self._test_qemu_ssh()
     testcase_path_name = os.path.basename(os.path.normpath(testcase_path))
     self.device.store(testcase_path, self.fuzzer.data_path())
 
     # TODO(flowerhack): Pass libfuzzer args (additional_args) here
     return_code = self.fuzzer.start(['repro', 'data/' + testcase_path_name])
     self.fuzzer.monitor(return_code)
-    self.fetch_and_process_logs_and_crash()
```

Comment:
Why call this self.process_logs_and_crash() at all if it will bail out instantly on the first line ?

Reply:
If we remove this, then can even remove the artifact_prefix check since we expect it to be always there.

Reply:
yep I realised this afterwards :) will clean this up